server/controllers/userInputController.ts

```diff
@@ -56,7 +49,7 @@ export const fetchUserInputList = async (req: Request, res: Response) => {
   try {
     // Retrieve the user ID from the request headers or use a mock ID in non-production environments
     const userId =
-      process.env.NODE_ENV !== 'production'
+      process.env.NODE_ENV === 'production'
         ? req.headers['x-user-id'] || req.headers['x-user-id']
         : MOCK_USER_ID;
```

Comment:
Just trying to understand the check here, if the environment is production, then we use req.headers or else we use mock user id?

Reply:
yup correct!